docs/guide/cx/introduction.qmd

```diff
@@ -0,0 +1,27 @@
+---
+title: "Introduction to Contributor Experience"
+sidebar: guide
+---
+
+What is contributor experience?
+
+Behind every successful open source project is a strong contributor community. 
+Yes, software must provide solutions. Ideally, should be a useful piece in a larger 
+solution to a larger problem, a part of a tool-chain. However, it’s the people, the 
+community, the culture of the community that determine the longevity of an open 
+source project. And when we talk about the culture of any given open source community, 
+what we really mean is the sum of:
+
+- values, hopefully all documented as policies and guidelines, 
+- technical infrastructure
+- and social infrastructure 
```

Comment:
Not sure if there's a grammatical best choice, but maybe
```suggestion
- technical infrastructure, and
- social infrastructure
```

Reply:
@melissawm After giving it further thought, I suggest we skip _and_ altogether.